buildlib/tools/builds.sh

```diff
@@ -448,6 +448,30 @@ then
 			'build_armclang')
 fi
 
+# Check if a specific test was requested as an argument
```

Comment:
IMO excessive comments, can summarize them all in 1 sentence

Reply:
done